reference/catalog.v3.yml

```diff
@@ -14602,10 +14602,7 @@ paths:
         - name: product_id
           in: query
           type: string
-          description: |-
-            A comma-separated list of ids of Products whose prices were requested. For example:
-            `?product_id=:id`
-            `?product_id:in=77,80,81`
+          description: 'A comma-separated list of ids of products whose variants were requested. For example:`?product_id=:id``?product_id:in=77,80,81`'
```

Comment:
description: 'A comma-separated list of IDs of products whose variants were requested. For example:`?product_id=:id``?product_id:in=77,80,81`'